test/azure-authenticator.js

```diff
@@ -1,9 +1,10 @@
 const assert = require('assert')
 const factory = require('../').factory
-
+const logger = require('../lib/logger')
 /* eslint-env node, mocha */
 
 describe('azure-chaos', () => {
+  beforeEach(() => logger.configure({ logImpl: () => null })) // noop by default
```

Comment:
nit: newline after

Reply:
further, this is inconsistent, please standardize to either all one line, or multi line